### ISSUE TITLE
react-ui <Dropdown /> not rendering correctly in Webkit browsers

### ISSUE THREAD

**Issue by moritzjacobs** [datocms/plugins-sdk#32]
Firefox:
![Bildschirmfoto 2024-09-05 um 14 08 21](https://github.com/user-attachments/assets/093f25d4-2858-4ed0-ae02-c8ffb0d9473d)

Safari:
![Bildschirmfoto 2024-09-05 um 14 08 25](https://github.com/user-attachments/assets/ebe82af1-45cc-45d0-8e79-1c1a897a62a3)

Reproduction repo: https://github.com/moritzjacobs/datocms-react-ui-dropdown-repro/ via `npx tmplr datocms/datocms-plugin-template`

```tsx
import { connect, RenderFieldExtensionCtx } from "datocms-plugin-sdk";
import "datocms-react-ui/styles.css";
import ConfigScreen from "./entrypoints/ConfigScreen";
import { render } from "./utils/render";
import {
  Canvas,
  Dropdown,
  Button,
  CaretUpIcon,
  CaretDownIcon,
  DropdownMenu,
  DropdownOption,
  DropdownSeparator,
} from "datocms-react-ui";

const Foobar = ({ ctx }: { ctx: RenderFieldExtensionCtx }) => {
  return (
    <Canvas ctx={ctx}>
      <Dropdown
        renderTrigger={({ open, onClick }) => (
          <Button
            onClick={onClick}
            rightIcon={open ? <CaretUpIcon /> : <CaretDownIcon />}
          >
            Options
          </Button>
        )}
      >
        <DropdownMenu>
          <DropdownOption onClick={() => {}}>Edit</DropdownOption>
          <DropdownOption disabled onClick={() => {}}>
            Duplicate
          </DropdownOption>
          <DropdownSeparator />
          <DropdownOption red onClick={() => {}}>
            Delete
          </DropdownOption>
        </DropdownMenu>
      </Dropdown>
    </Canvas>
  );
};

connect({
  renderConfigScreen(ctx) {
    return render(<ConfigScreen ctx={ctx} />);
  },
  manualFieldExtensions() {
    console.log("manualFieldExtensions");
    return [
      {
        id: "foobar",
        name: "Foobar: Just a test",
        type: "editor",
        fieldTypes: ["string"],
      },
    ];
  },
  renderFieldExtension(fieldExtensionId: string, ctx: RenderFieldExtensionCtx) {
    switch (fieldExtensionId) {
      case "foobar":
        return render(<Foobar ctx={ctx} />);
    }
  },
});
```

… using the "Basic example" from [here](https://www.datocms.com/docs/plugin-sdk/dropdown).

Thanks for your support!

**Comment by arcataroger** [comment]
Thank you for the report! Let's keep this issue open, but try to consolidate discussion (for now) in the forum: https://community.datocms.com/t/react-ui-dropdown-not-rendering-correctly-in-webkit-browsers/5280/2

I'll close this issue once the bug is fixed.

**Comment by stefanoverna** [comment]
Should be fixed in v1.1.1

datocms-plugin-sdk@1.1.1
datocms-react-ui@1.1.1

**Comment by moritzjacobs** [comment]
That was fast!
I can confirm it works in my repro now as well. Thank you!
